packages/cli/src/lib/config.ts

```diff
@@ -38,7 +38,7 @@ export async function loadCliConfig(options: Options) {
   const configTargets: ConfigTarget[] = [];
   options.args.forEach(arg => {
     if (!isValidUrl(arg)) {
-      configTargets.push({ path: paths.resolveTarget(arg) });
+      configTargets.push({ path: paths.resolveTargetRoot(arg) });
```

Comment:
Sup with this change?

Reply:
`paths.resolveTarget` resolves against cwd, `paths.resolveTargetRoot` against the root of the mono-repo, just so that you don't need to reverse-engineer the code itself. And `cwd` is the package directory. If you'd run from the top-level, you'd have to pass relative paths with `../../` like we do for catalog entries in `app-config.yaml` ;-)

Reply:
Yep, I know how it works ;) To be clear I'm not really opposed to this change as such, it makes sense, it's just that it's very much a breaking change. There are a lot of `--config ../../app-config.production.yaml` etc. out there. We'd need to ship this in a more backwards compatible way I'd say.

For now I think it's best if we simply resolve the config paths upfront and pass resolved paths to the upstream builds, either as absolute paths, or relative. If you want to spend some time finding a nice way to ship this change than that's alright too, but that is likely fairly tricky.

Reply:
I was wondering about this, too. What do you think about doing

```typescript
    if (!isValidUrl(arg)) {
      let path = paths.resolveTarget(arg);
      if (!pathExistsSync(path)) {
        path = paths.resolveOwnRoot(arg);
      }
      configTargets.push({ path });
    }
```

That way, paths local to app and to the root would work?

Reply:
Figured I can also just push it if I have it written already. It's easy to undo if that's not the right way.

Reply:
Hmm, tbh I think it would be best to just forward the resolved config path for now, and not include this change. The change actually has fairly high impact and I wouldn't want to ship it strait into mainline on Tuesday. We'd also need to look over all other places that config is loaded to make sure it all behaves the same way.

Ok with skipping this for now and leaving it for a potential future change?

Reply:
Yeah, that's OK, but I won't get to it this week.

Reply:
Like so?